src/tests/ftest/security/cont_get_acl.yaml

```diff
@@ -0,0 +1,23 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - server-A
+  test_clients:
+    - client-B
+timeout: 100
+server_config:
+  name: daos_server
+pool:
+  control_method: dmg
+  mode: 511
+  scm_size: 2G
+  name: daos_server
+container:
+  control_method: daos
+  type: POSIX
+
+out_filename:
```

Comment:
Are these invalid file names? If so you might want to use invalid_out_filename or something. You have vAliD.txt, so I can't tell whether these are invalid names.

Reply:
They are valid filenames, I wanted to make them somewhat abnormal filenames to test the fact that the file should still be created regardless of the name. I'll make the yaml file tag more descriptive.